src/include/OpenImageIO/imagebufalgo.h

```diff
@@ -790,6 +790,68 @@ bool OIIO_API warp (ImageBuf &dst, const ImageBuf &src, M33fParam M,
 /// @}
 
 
+/// @defgroup st_warp (st_warp: warp an image using per-pixel st coordinates)
```

Comment:
Is there anything I need to do to update the docs besides adding the Doxygen comments here?

Reply:
No, that's probably enough for the IBA part.